Order.Management/Circle.cs

```diff
@@ -10,7 +10,7 @@ class Circle : Shape
         public Circle(int red, int blue, int yellow)
         {
             Name = "Circle";
-            base.Price = circlePrice;
```

Comment:
call to "base" could be removed as this call is redundant, as constructor of derived class implicitly calls the constructor for the base class